Madness/Parser.swift

```diff
@@ -1,5 +1,9 @@
 //  Copyright (c) 2014 Rob Rix. All rights reserved.
 
+public struct ResultResult<T, Error : ErrorType> {
+	public typealias t = Result<T, Error>
+}
```

Comment:
Also, this sucks. But it's necessary for now.

Reply:
lol